models/fivetran_log__connector_status.sql

```diff
@@ -114,6 +130,7 @@ connector_recent_logs as (
         connector_health.connector_type,
         connector_health.destination_id,
         connector_health.connector_health,
+        connector_health.last_successful_sync_completed_at,
         connector_health.last_sync_started_at,
         connector_health.last_sync_completed_at,
```

Comment:
is `last_sync_completed_at` valuable to have anymore now that there is `last_successful_sync_completed_at`? 

or like should we change the name to `last_sync_attempt_completed_at` to better differentiate those

Reply:
I was struggling with this one. I wanted to remove it, but in terms of the connector itself a sync can complete and not succeed. So I could see it being valuable to see the last successful sync vs the last completed sync to see if your connector has just been failing for the last few days.

Reply:
I feel we should keep it, but that is why I made sure to add the verbiage to the yml to highlight that it doesn't indicate a successful sync. What are your thoughts?

Reply:
actually yah i agree....like a sync can be partially successful right? could be worth it to know if some of your tables got written to while others did not

Reply:
and yeah i think your original yml description may highlight the differences enough